pyartcd/pyartcd/util.py

```diff
@@ -310,3 +310,232 @@ async def sync_images(version: str, assembly: str, operator_nvrs: list,
             doozer_data_path=doozer_data_path,
             doozer_data_gitref=doozer_data_gitref
         )
+
+
+def default_release_suffix():
+    """
+    Returns a release suffix based on current timestamp
+    E.g. "201901011200.?
+    """
+
+    return f'{datetime.strftime(datetime.now(), "%Y%m%d%H")}.p?'
```

Comment:
The timestamp should include minute:

```suggestion
    return f'{datetime.strftime(datetime.now(), "%Y%m%d%H%M")}.p?'
```

Reply:
Fixed in a82d6f286dc689606ee41ebb42e9d87fc666a59c